homeworks/igor.orel_igor0806/GamesProject/about.html

```diff
@@ -0,0 +1,342 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+        >
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>About</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+
+        <!-- Header  -->
+        <header class="intro">
+            <div class="nav">
+                <div class="logo">
+                    <img src="Icon/logo.svg" alt="img">
+                </div>
+
+                <div class="nav-menu">
+                    <ul class="link_list">
+
+                        <li class="link">
+                            <a class="nav-link" href="./index.html">Games</a>
```

Comment:
This page should be an index.html. Another page - games.html. 
Now, they are mismatched, and on click on Games link, you are going to About page.
Please update the names and links in the header.